devtools/server/actors/replay/connection.js

```diff
@@ -542,7 +544,57 @@ class Recording extends EventEmitter {
   }
 }
 
-function canRecordUrl(url) {
+function checkShouldValidateUrl() {
+  if (gShouldValidateUrl !== null) {
+    return Promise.resolve(gShouldValidateUrl)
+  } else {
+    return fetch(getAPIServer(), {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${ReplayAuth.getReplayUserToken()}`,
+      },
+      body: JSON.stringify({
+        query: `
+          query GetOrgs {
+            viewer {
+              workspaces {
+                edges {
+                  node {
+                    isOrganization
+                    settings {
+                      features
+                    }
+                  }
+                }
+              }
+            }
+          }
+        `,
+      })
+    }).then(resp => resp.json()).then(resp => {
+      if (resp.errors) {
+        throw new Error("Unexpected error checking Replay user permissions");
+      }
+
+      const workspaces = resp.data.viewer.workspaces.edges;
+      gShouldValidateUrl = !workspaces ? false : workspaces.some(w => {
+        if (w.node.isOrganization) {
+          const {allowList, blockList} = w.node.settings && w.node.settings.features && w.node.settings.features.recording || {};
```

Comment:
`w.node?.settings?.features?.recording || {}`

Reply:
Didn't know if I could get away with those in gecko so I went old school

Reply:
Hey, it works! Updated 😎